save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/TopBar.kt

```diff
@@ -180,13 +181,21 @@ fun topBar() = fc<TopBarProps> { props ->
                         set("aria-expanded", "false")
                     }
 
-                    span("mr-2 d-none d-lg-inline text-gray-600") {
-                        +(props.userInfo?.name ?: "")
-                    }
-
-                    fontAwesomeIcon {
-                        attrs.icon = "user"
-                        attrs.className = "fas fa-lg fa-fw mr-2 text-gray-400"
+                    div("row") {
+                        div {
+                            span("mr-2 d-none d-lg-inline text-gray-600") {
+                                +(props.userInfo?.name ?: "")
+                            }
+                            fontAwesomeIcon {
+                                attrs.icon = "user"
+                                attrs.className = "fas fa-lg fa-fw mr-2 text-gray-400"
+                            }
+                        }
+                        if ((props.userInfo?.globalRole ?: Role.NONE).priority >= Role.ADMIN.priority) {
+                            small("text-gray-400 text-justify") {
+                                +props.userInfo?.globalRole?.formattedName!!
```

Comment:
Well, it is not null because all the time you are logged in you can see your nickname at the top bar. The point is that `globalRole` in userInfo is null during the very first render. Seems like need to think of a solution.

Reply:
Yes, I didn't see the condition on `globalRole`. And if `globalRole` is null, we won't get inside this condition either. But maybe it will be more readable, if you save `props.userInfo?.globalRole` into a variable to emphasize its non-nullability inside the condition

Reply:
Ok, done.

Reply:
That's only part of what I meant; now you can use `val globalRole` inside `if` body as well, and you won't need `!!`

Reply:
Of course, yes, sorry